wpeverest/user-registration
# Added - Dev Compatibility for Checkout Page frontend validation.

### All Submissions:



### Changes proposed in this Pull Request:

This Dev Compatibility implements Frontend validation of fields on Checkout Page from Woocommerce.

UR Woocommerce Dependency - https://github.com/wpeverest/user-registration-woocommerce/pull/40

### How to test the changes in this Pull Request:

1. User Registration > Add New > Create a Form > Drag many fields > Set validations from Advance Settings ( activate File Upload and file upload field ).
2. User Registration Settings > Woocommerce > Sync  UR fields on Checkout Page.
3. Create a product from Woocommerce > sync UR fields from UR Form tab.
4. Open in incognito mode Product page. Verify if frontend and backend validation is working or not.
5. Add to Cart and goto Checkout page. Check create user.
6. Verify if frontend and backend validation is working or not on synced fields.

### Types of changes:




### Other information:




### Changelog entry

> Added - Dev Compatibility for Checkout Page frontend validation.
